weltyt.py

```diff
@@ -0,0 +1,23 @@
+import sqlite3
+
+
+print("Welcome to tyton")
+conn = sqlite3.connect("TYTDB.db")
+
+cur = conn.cursor()
+base = open("TYTDB.sql")
+
+for line in base:
+        cur.execute(line)
+
+
+
+conn.commit()
+
+for row in cur.execute('SELECT * FROM resources'):
```

Comment:
Those two prints were helpful during testing, but I don't think they are a part of the ticket itself.